packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -122,6 +123,13 @@ export async function fetchHelper(
 					{ driverVersion },
 				);
 			}
+
+			if (!response.ok && response.redirected) {
+				throw new RequestRedirectionError("ODSP fetch error: Redirection", response.url, {
```

Comment:
I think this could interrupt our other flows. This is just a fetch util and we don't create these types of errors here. They are generated in "throwOdspNetworkError" utility.